src/adapters/PeopleJSONAdapter.js

```diff
@@ -22,14 +22,27 @@ import {Observable} from 'rxjs';
  * }
  */
 
-/*
- * Implements the PeopleAdapter interface with a JSON object as its datasource. See @PeopleJSON
+/**
+ * Implements the PeopleAdapter interface with a JSON object as its datasource.
+ *
+ * @see {@link PeopleJSON} for example datasource.
+ * @class
+ * @implements {PeopleAdapter}
  */
 export default class PeopleJSONAdapter extends PeopleAdapter {
-  constructor(datasource) {
-    super(datasource);
+  /**
+   * Returns an observable that emits person data of the default user.
+   *
+   * @returns {Observable.<Person>}
+   * @memberof PeopleJSONAdapter
+   */
+  getMe() {
+    const defaultUserID = 'default';
 
-    this.getPerson = this.getPerson.bind(this);
+    return Observable.create((observer) => {
```

Comment:
no `observer.error`?

Reply:
No, the current user should always exist